### DISCUSSION TITLE
Classes Order is changed on Build

### DISCUSSION THREAD

**Discussion by PRigter** [parcel-bundler/parcel#5423] [General]
Hi all, 

It seems that on **build**, Parcel changes the classes order. 

**I'm using Semantic UI, in development, this is the correct class order:**

![image](https://user-images.githubusercontent.com/30890518/101139030-d8cec580-3608-11eb-8090-f238f35be656.png)

**After Build, looks like that Parcel is changing classes to Alphabetical Order!** - That messes up all the layout 😭 

![image](https://user-images.githubusercontent.com/30890518/101139199-1cc1ca80-3609-11eb-89c1-40982cb26335.png)

Is there a way to prevent this behavior ? 🤔 

Thanks!

**Comment by mischnic** [comment]
This is probably caused by htmlnano's sortAttributesWithLists rule: https://github.com/posthtml/htmlnano#sortAttributesWithLists

Does adding a `.htmlnanorc` file with this fix it?: 
```
{
	"sortAttributesWithLists": false
}
```
